app/run.py

```diff
@@ -53,6 +54,14 @@
     debug_logging
 )
 
+# Setup message handlers
+message_parsers = [
+#    NoMatchParser,
+#    MatchExceptionParser,
```

Comment:
There are no automated tests of the execution. Those test parsers were useful when building support to make sure we handled cases that aren't easy to test without having intentionally broken parsers. I've removed them from `app/run.py` but left them in the parsers directory in case they're useful for adding features in the future.